src/PowerShellEditorServices/Services/Analysis/AnalysisService.cs

```diff
@@ -941,43 +415,75 @@ private static DiagnosticSeverity MapDiagnosticSeverity(ScriptFileMarkerLevel ma
         {
             switch (markerLevel)
             {
-                case ScriptFileMarkerLevel.Error:
-                    return DiagnosticSeverity.Error;
+                case ScriptFileMarkerLevel.Error:       return DiagnosticSeverity.Error;
+                case ScriptFileMarkerLevel.Warning:     return DiagnosticSeverity.Warning;
+                case ScriptFileMarkerLevel.Information: return DiagnosticSeverity.Information;
+                default:                                return DiagnosticSeverity.Error;
+            };
+        }
 
-                case ScriptFileMarkerLevel.Warning:
-                    return DiagnosticSeverity.Warning;
+        private static Hashtable GetCommentHelpRuleSettings(string helpLocation, bool forBlockComment)
+        {
+            return new Hashtable {
+                { "Rules", new Hashtable {
+                    { "PSProvideCommentHelp", new Hashtable {
+                        { "Enable", true },
+                        { "ExportedOnly", false },
+                        { "BlockComment", forBlockComment },
+                        { "VSCodeSnippetCorrection", true },
+                        { "Placement", helpLocation }}
+                    }}
+                }};
+        }
 
-                case ScriptFileMarkerLevel.Information:
-                    return DiagnosticSeverity.Information;
+        #region IDisposable Support
+        private bool disposedValue = false; // To detect redundant calls
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this initialization to 'disposedValue', the compiler will do that for you.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4961943)